pipelines/tekton/validate.go

```diff
@@ -30,7 +30,7 @@ func validatePipeline(f fn.Function) error {
 			return ErrRuntimeRequired
 		}
 
-		if f.Runtime == "go" || f.Runtime == "rust" {
+		if f.Runtime == "rust" {
```

Comment:
Maybe we could remove this `if` completely since following `if` check if there are additional buildpacks.

Reply:
I think it might be better to keep it here, to explicitly tell user about not supported runtime.

Reply:
But the issue with Go and Rust is that they use custom buildpacks.
That problem is detected in following `if`.

What this `if` does is it prevents users from building Go and Rust apps even if user provides custom builder that would work all right.

Reply:
@zroubalik @Shashankft9 what about just printing warning here "Go/Rust default builder do not support in cluster build" instead of returning an error? Could we accept that?

Reply:
> @zroubalik @Shashankft9 what about just printing warning here "Go/Rust default builder do not support in cluster build" instead of returning an error? Could we accept that?

@Shashankft9 Can you add a simple warning here that Go and Rust default builders do not support in-cluster builds?